internal/featuretoggle/monitoring_stack_feature_toggle.go

```diff
@@ -27,20 +32,89 @@ func (m *MonitoringStackFeatureToggle) Name() string {
 }
 
 func (m *MonitoringStackFeatureToggle) GetFeatureToggleIdentifier() string {
+	// TODO: should this be changed to "MONITORING_STACK"?
```

Comment:
It was previously decided across the team that we want to expose all feature-toggles, at their first implementation, under the feature toggle named "EXPERIMENTAL_FEATURES" so as to mark them as features which we are experimenting with and they're not usable in prod. 
This includes the monitoring_stack feature toggle as well.

Unless, we are in a very tight situation where we don't want to enable "EXPERIMENTAL_FEATURES" in a certain environment yet ship that feature in a toggleable-in-runtime manner, only then, we should expose it under a different feature toggle identifier such as "MONITORING_STACK" 

Also, we don't want to bloat the runtime configuration of addon-operator (saas-addon-operator) with a list of feature-toggles because the more feature-toggles we are dealing with IN RUNTIME, the worse it is. Feature toggles should have a short life, either they end up getting promoted as a core feature or they're removed in small amount of time. 

Therefore, we intend to specify all such feature-toggles under the "EXPERIMENTAL_FEATURES" feature toggle.

Reply:
Hi Yash, sorry for the late response. Ohh I didn't know that. Addons plug and play is enabled via its own feature toggle https://github.com/openshift/addon-operator/blob/main/internal/featuretoggle/addons_plug_and_play_feature_toggle.go#L16. Should that also be switched to `EXPERIMENTAL_FEATURES` then?

Reply:
So, the EXPERIMENTAL_FEATURES feature toggle works only on integration and not even on stage+production. So if ADDONS_PLUG_AND_PLAY is not, yet, required on stage/production you can go ahead with EXPERIMENTAL_FEATURES. Otherwise, you can just continue with ADDONS_PLUG_AND_PLAY itself :)